servers/quarkus-server/src/test/java/org/projectnessie/server/TestRest.java

```diff
@@ -167,23 +159,25 @@ void referenceNames(String refNamePart) throws NessieNotFoundException, NessieCo
     String branchName = "branch" + refNamePart;
     String branchName2 = "branch2" + refNamePart;
 
-    String someHash = tree.getReferenceByName("main").getHash();
+    Reference main = tree.getReferenceByName("main");
+    String someHash = main.getHash();
 
     Reference createdTag = tree.createReference(Tag.of(tagName, someHash));
     assertEquals(Tag.of(tagName, someHash), createdTag);
-    Reference creadedBranch1 = tree.createReference(Branch.of(branchName, someHash));
-    assertEquals(Branch.of(branchName, someHash), creadedBranch1);
-    Reference creadedBranch2 = tree.createReference(Branch.of(branchName2, someHash));
-    assertEquals(Branch.of(branchName2, someHash), creadedBranch2);
+    Reference createdBranch1 = tree.createReference(Branch.of(branchName, someHash));
+    assertEquals(Branch.of(branchName, someHash), createdBranch1);
+    Reference createdBranch2 = tree.createReference(Branch.of(branchName2, someHash));
+    assertEquals(Branch.of(branchName2, someHash), createdBranch2);
 
     Map<String, Reference> references = tree.getAllReferences().stream()
         .filter(r -> "main".equals(r.getName()) || r.getName().endsWith(refNamePart))
         .collect(Collectors.toMap(Reference::getName, Function.identity()));
-    assertThat(references.values(), containsInAnyOrder(
-        referenceWithNameAndType("main", Branch.class),
-        referenceWithNameAndType(tagName, Tag.class),
-        referenceWithNameAndType(branchName, Branch.class),
-        referenceWithNameAndType(branchName2, Branch.class)));
+
+    assertThat(references).containsAllEntriesOf(
```

Comment:
Looks like this one removes the assertions on the type ("instanceof")

Reply:
fixed